src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -81,37 +86,38 @@
 
 <kd-help-section>
   <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-             ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+               ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
     Expose service externally
   </md-checkbox>
 </kd-help-section>
 
+<!-- advanced options -->
 <div ng-show="ctrl.isMoreOptionsEnabled()">
   <kd-help-section>
-    <md-input-container>
+    <md-input-container class="md-block kd-deploy-block">
       <label>Description</label>
       <textarea ng-model="ctrl.description"></textarea>
     </md-input-container>
-    <kd-user-help>
+    <kd-user-help class="kd-extra-padding-top">
```

Comment:
I would suggest to remove. We should aim for a consistent layout concept. I would only make things inconsistent in case a ui expert asks for it.